CONTRIBUTING.md

```diff
@@ -1,29 +1,67 @@
 # Contributing to Coq
 
-Thank you for your interest in contributing to Coq! There are many ways to contribute, and we appreciate all of them.
+Thank you for your interest in contributing to Coq! There are many ways to
+contribute, and we appreciate all of them.
```

Comment:
Maybe this is a good moment to mention the Code of Conduct?

Reply:
Good point!